Config/Filter/Type/Background.php

```diff
@@ -43,9 +43,9 @@
 
     /**
      * @param string      $name
-     * @param string|null $color
-     * @param string|null $transparency
-     * @param string|null $position
+     * @param string|null $color        background color HEX value
+     * @param string|null $transparency possible values 0..100
+     * @param string|null $position     position of the input image on the newly created background image. Valid values: topleft, top, topright, left, center, right, bottomleft, bottom, and bottomright
```

Comment:
Maybe we can use value objects/enum here also.

Reply:
@Koc for all parameters or for specific one ?